addons/web/models/ir_http.py

```diff
@@ -33,7 +33,7 @@
 class Http(models.AbstractModel):
     _inherit = 'ir.http'
 
-    bots = ["bot", "crawl", "slurp", "spider", "curl", "wget", "facebookexternalhit", "whatsapp", "trendsmapresolver", "pinterest", "instagram"]
+    bots = ["bot", "crawl", "slurp", "spider", "curl", "wget", "facebookexternalhit", "whatsapp", "trendsmapresolver", "pinterest", "instagram", "google-pagerenderer", "preview"]
```

Comment:
If you are interested, there exists an [extension](https://webextension.org/listing/useragent-switcher.html) that regroups a lot of UA and has a bot section. You can also set a custom one with it to test.